tests/integration/replication.tcl

```diff
@@ -398,7 +398,9 @@ foreach testType {Successful Aborted} {
             # Set master and replica to use diskless replication on swapdb mode
             $master config set repl-diskless-sync yes
             $master config set repl-diskless-sync-delay 0
+            $master config set save ""
```

Comment:
i'm not sure why we needed that, since we did have this:
```
                    # Speed up shutdown
                    $master config set rdb-key-save-delay 0
```
we had it only in the aborted path, but that's also the only one that set it to anything other than 0 in the first place.
can you share some time measurements of before and after?
in any case, these lines (which i just quoted) are now no longer needed.

one more thing, i didn't bother to verify that the new numbers (delay and number of keys) you use are safe to avoid failures due to race conditions and slowness.
please make sure they're causing the replication to be long enough so that we're sure we'll be able to interrupt it in the middle (i.e some 5 or 10 seconds should be ok)

Reply:
The new numbers make replication longer. Master has twice the keys (from 500 to 1000).
The only impact I can see from the number of keys in replica is rdb flushing time to disk (only on valgrind test or when I use absurd amount of keys in my machine).

About not needing:
`$master config set save ""`

Aren't the 1000 keys in master going to be flushed to disk when we terminate master server after Aborted path?

Reply:
Can you run that valgrind again so we can see if the "Waiting for process nnn to exit" disappeared from all these tests after the `set save ""` was added?

> can you share some time measurements of before and after?

Changing the replica keys to 20_000_000 changed total running time of runtest-moduleapi from 10s to 30s here, and started showing `Waiting for process n to exit` about 5 times.
But with these same 20 million keys and `set save ""` it's back to the 10 to 11s and there's no `Waiting for process`

Reply:
i meant that we no longer need this, didn't suggest to remove `config set save ""`
```tcl
                    # Speed up shutdown
                    $master config set rdb-key-save-delay 0
```

i've re-triggered the tests (same links).

ideally we should tune it so that the "successful" test takes about a second or two,
and the "aborted" test sets delays that would take at least 5 (or better yet 20) seconds, but since we abort it rather early, it also takes a second or two.

Reply:
Successful replication here takes about 1.1s.
Aborted (if not interrupted) about 11s, and interrupted as it is 1.2s

Reply:
great! that's what we wanted.
thank you!